dev/tests/functional/tests/app/Magento/Catalog/Test/Constraint/AssertCategoryProductsGridFilter.php

```diff
@@ -0,0 +1,2083 @@
+
```

Comment:
Looks like it's mistake

Reply:
@ihor-sviziev what's a mistake?

Reply:
Just look at file contains. It looks like HTML

Reply:
Uhh wow how did that get there. Ok I'll fix.